test/dataset/test_pandas.py

```diff
@@ -15,7 +15,7 @@
 import numpy as np
 import pytest
 
-from gluonts.dataset import dataframe
+from gluonts.dataset import pandas
```

Comment:
Now this looks a bit weird! Maybe we should rename this to `pandas_dataset` , or just `pd` -- oh wait!

Let's leave it like that.